### PR TITLE
Allow setting permalink zoom with map_zoom param

### PR DESCRIPTION
**Pull request by svareg**
Allow to set the zoom level with the parameter **map_zoom** in the URL for a WFS-Permalink.

### CODE REVIEW

File: src/statemanager/WfsPermalink.js [hunk 1]
```diff
@@ -252,7 +260,7 @@ WfsPermalinkService.prototype.issueRequest_ = function(wfsType, filter, map, sho
     // zoom to features
     const size = map.getSize();
     if (size !== undefined) {
-      const maxZoom = this.pointRecenterZoom_;
+      const maxZoom = zoomLevel || this.pointRecenterZoom_;
```
Comment (sbrunner) [inline src/statemanager/WfsPermalink.js]: Don't work with zoomLevel == 0...

**Comment by sbrunner** [comment]
@fredj does it look good to you?

File: src/statemanager/WfsPermalink.js (hunk 1 above)
Comment (pfirpfel) [reply src/statemanager/WfsPermalink.js:263]: @svaabs Can you change it to this:
`const maxZoom = zoomLevel !== undefined ? zoomLevel : this.pointRecenterZoom_;`

**Comment by fredj** [comment]
looks good to me

**Comment by jwkaltz** [comment]
@sbrunner review OK? (fredj wrote that it is OK for him)

**Comment by sbrunner** [comment]
+1, merging :-)

**Pull request merged**